testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -345,17 +345,17 @@ def test_verbose_progressbar(u, capsys):
 def test_verbose_progressbar_run(u, capsys):
     FrameAnalysis(u.trajectory).run(verbose=True)
     _, err = capsys.readouterr()
-    expected = u'100%|██████████| 98/98 [00:00<00:00, 8799.49it/s]'
```

Comment:
I think the reason why the full output was there originally was to give devs an idea what it looks like even if other parts could not be reliably tested.

Reply:
Unfortunately the py3.13 behaviour changed here and I can't work out exactly why - I'm not sure if there's a better way to deal with this without having an explicit python version check :/